android/BOINC/app/src/main/java/edu/berkeley/boinc/ProjectsFragment.java

```diff
@@ -158,7 +158,7 @@ private void updateData(ArrayList<Project> latestRpcProjectsList, AcctMgrInfo ac
 		for(int x = 0; x < data.size(); x++) {
 			if(data.get(x).isMgr) {
 				mgrIndex = x;
-				continue;
+				break;
```

Comment:
The question here is: can there be more than one AM entry? If so, which one should be recorded, the last (original) or the first (your version)? Given the context I tend to accept your change.

Reply:
I think changing `continue` to `break` will break the logic here. I'd rather just remove `continue` here.
BTW, I don't fully understand the purpose of this function at all.
In case of mane AMs present in the list current implementation will overwrite the last AM entry.
New proposed implementation will overwrite first only.
I think more detailed investigation is necessary here to understand the purpose and desired behavior of this function

Reply:
Ok, so the decision is to not optimize this right now, but to flag this as an open question (providing how we think it should be written). FTR: removing the `continue` doesn't change the semantics.